stunnel/dev_https

```diff
@@ -0,0 +1,10 @@
+pid=
+
+cert = stunnel/stunnel.pem
+foreground = yes
+output = stunnel.log
```

Comment:
And/or change this to `output = stunnel/stunell.log` so all the `stunnel` files are kept (and ignored) together?